src/registry/registry.ts

```diff
@@ -83,6 +98,18 @@ export function registerExpressionHelper(name: string, fn: Function) {
   getFormsStore().expressionHelpers[name] = fn;
 }
 
+export function registereformSchemaTransformers(registration: FormScemaTransformerRegistration) {
+  const store = getFormsStore();
+  const existingIndex = store.formTransformers.findIndex((reg) => reg.name === registration.name);
+
+  if (existingIndex !== -1) {
+    // If registration with the same name exists, override it
+    store.formTransformers[existingIndex] = registration;
+  } else {
+    store.formTransformers.push(registration);
+  }
+}
```

Comment:
This seems to imply that the transformers are stored in array yet they seem to be stored in a map. I would simply:

```ts
getFormsStore().formTransformers[registration.name] = registration;
```

Reply:
Hi @samuelmale 
If I got you right, it's actually an array.
This is how the FormsRegistryStoreState looks like:
```
  export interface FormsRegistryStoreState {
  controls: CustomControlRegistration[];
  fieldValidators: ComponentRegistration<FieldValidator>[];
  fieldSubmissionHandlers: FieldSubmissionHandlerRegistration[];
  postSubmissionActions: ComponentRegistration<PostSubmissionAction>[];
  dataSources: ComponentRegistration<DataSource<any>>[];
  expressionHelpers: Record<string, Function>;
  formSchemaTransformers: FormSchemaTransformerRegistration[];
}
```

I followed the same logic for submission handlers, the only difference being that for transformers we override
```
export function registerFieldSubmissionHandler(registration: FieldSubmissionHandlerRegistration) {
  getFormsStore().fieldSubmissionHandlers.push(registration);
}
```